ios/FluentUI.Demo/FluentUIDemoTests/NotificationViewTest_SwiftUI.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import XCTest
+
+class NotificationViewTestSwiftUI: BaseTest {
+    override var controlName: String { "NotificationView" }
+
+    override func setUpWithError() throws {
+        try super.setUpWithError()
+        continueAfterFailure = false
+        app.buttons["Show"].firstMatch.tap()
+    }
+
+    // launch test that ensures the demo app does not crash
+    func testLaunch() throws {
+        XCTAssertTrue(app.exists)
```

Comment:
For the SwiftUI pages, you need to use `NSPredicate` (the titles contain other text like Fluent 2 or SwiftUI) and they run fine by themselves but for some reason, when you try to run all the tests together, they'll cause failures. If we eventually add these checks to the GitHub, I worry that this might cause them to be false negatives. I'm looking into why they're failing so I just pushed the working version.